### PR TITLE
4.19.0 prep

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by tphoney** [comment]
merge if you think this is good to go.. we should speak about the compare 
@eputnam

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,28 @@
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org).
 
+## Supported Release 4.19.0
+### Summary
+
+This release adds new functions and better documentation/fixes for existing functions. With a noteworthy fixes to file_line.
```
Comment (eputnam) [inline CHANGELOG.md]: Maybe something more like "This release adds new functions and better documentation/fixes for existing functions with a noteworthy fix to file_line."

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,28 @@
...
+
+#### Added
+- Add validate_domain_name function
+- Add the round function
+- Add type for MAC address
+- (MODULES-4908) adds support for sensitive data type to pw_hash
```
Comment (eputnam) [inline CHANGELOG.md]: we've been trying to put the ticket numbers at the end and including a link for each on public tickets.

**Comment by tphoney** [comment]
@hunner  merge ?

**Pull request merged**